semantic_model_generator/validate/context_length.py

```diff
@@ -0,0 +1,11 @@
+_MODEL_CONTEXT_LENGTH = 7000  # We use 7k so that we can reserve 1k for response tokens.
```

Comment:
Shouldn't we account for instruction tokens in the prompt?

Reply:
This is pretty loose as is. Instruction tokens, even for llama3, is only ~20 tokens. 

I can add an additional buffer here though!